### PR TITLE
technical crawler added

### PR DESCRIPTION
**Pull request by veljkovic**

### CODE REVIEW

**Comment by ximenina** [comment]
Hello,

Thanks for your PR. Unfortunately, I haven't been able to merge it since the [database loader ](https://github.com/MorphDiv/TeDDi_sample/tree/master/Database) outputs errors when parsing the files: 

- One of these errors is related to the meta-data header:

In the text files, you have this field: `# ISO_6393:`  instead of `# iso639_3:`.

- There is another error. However, I haven't been able to spot the origin of this problem (maybe in the filenames or the  folder structure).

In the meantime, is it possible for you to run this test and try to identify and correct the problem? ([Documentation](https://github.com/MorphDiv/TeDDi_sample/tree/master/Database))

Basically, you have to put the new corpora that you want to test under [tests/Corpus](https://github.com/MorphDiv/TeDDi_sample/tree/master/Database/tests/Corpus). Then, you run the database script `python3 load-database.py`. If the new texts are fully compatible there should be no errors when running this script.



- Finally, could you move your `report_technical.csv`to the Reports folder? (just out of consistency)

**Comment by veljkovic** [comment]
Hi,
Thank you for your response.
Regarding comments:
1. When it comes to meta-data `ISO_6393` field is defined in other crawlers as such (for example gutenberg_crawler.py, opus_opensubtitles_crawler.py). I can change it to ` iso639_3` if it's needed, I kept it the same as in other crawlers for consistency.
2.  Sure, I'll try to find where is the problem with tests not passing, if you have some more specific directions it would be appreciated. Firstly, I noticed that `'technical'` genre doesn't exist in ` GENRE_BROAD` variable, not sure if that is something that I should change or if it's managed by you. Also, there are two new languages added via this PR (`'mg': ['Malagasy_plt', 'plt', 'Malagasy', 'Plateau Malagasy', 'Latn'], 'om': ['Oromo_Harar_hae', 'hae', 'Oromo (Harar)', 'Eastern Oromo', 'Latn']`), again not sure if that should be managed by you or me, since it is related to database maintenance.
3. Sure, I will move it to Reports, along with other changes.

**Comment by ximenina** [comment]
Thanks for this info. That gives me a clue about where to look for the error.  I will check on that and keep you updated

**Comment by bambooforest** [comment]
IIRC correctly I used iso639_3 because it would be easily mergable with
other resources like Glottolog.

Wrt to technical, if that’s a new category then I would extend the constant
to include it for testing.

**Comment by ximenina** [comment]
The problem was indeed the lack of the `technical` category in `models.py`. I'll change that (and the new languages addition)

@veljkovic Still the change to `iso639_3` is required so the database can parse it. Thanks!

**Comment by veljkovic** [comment]
Hi @ximenina,
I moved the technical report to the Reports folder and changed the heading to iso639_3 in the technical crawler and files. Hope everything works as expected now.

**Comment by ximenina** [comment]
Database loader runs OK
